app/views/user_groups/_form.html.slim

```diff
@@ -1,2 +1,3 @@
 = form.input :name
 = form.input :description
+= form.association :users, label_method: :to_s, collection: User.all
```

Comment:
we should probably order these somehow. Since we do not have first name and last name for all users, perhaps by email address?